Sources/SwiftTranslate/Bootstrap/SwiftTranslate.swift

```diff
@@ -61,6 +61,12 @@ struct SwiftTranslate: AsyncParsableCommand {
     )
     private var requestRetry: Int = 1
 
+    @Option(
+        name: [.customLong("timeout")],
+        help: "Timeout interval for OpenAI API requests"
```

Comment:
@FabrizioBrancati should probably be `Timeout interval for API requests`

Reply:
Done 👌